charts/tfy-karpenter-config/templates/karpenter-inferentia-default-ec2nodeclass.yaml

```diff
@@ -1,15 +1,17 @@
-apiVersion: karpenter.k8s.aws/v1alpha1
-kind: AWSNodeTemplate
+apiVersion: karpenter.k8s.aws/v1beta1
+kind: EC2NodeClass
 metadata:
   name: inferentia-default
   namespace: karpenter
 spec:
   instanceProfile: {{ .Values.karpenter.instanceProfile }}
-  subnetSelector:
-    kubernetes.io/cluster/{{ .Values.cluster.name }}: shared
-    subnet: private
-  securityGroupSelector:
-    karpenter.sh/discovery: {{ .Values.cluster.name }}
+  subnetSelectorTerms:
+    - tags:
```

Comment:
Let's add support for additional tags 
We can add an option to pass support for additional tags similar to 
```
{{- range $key, $val := .Values.karpenter.inferentiaDefaultNodeTemplate.extraTags }}
```